src/libs/elektra/keyset.c

```diff
@@ -1254,6 +1254,83 @@ elektraCursor ksFindHierarchy (const KeySet * ks, const Key * root, elektraCurso
 	return it;
 }
 
+/**
+ * Retrieves all Keys from KeySet @p ks that are below or at @p root
+ *
+ * This function is similar to ksCut().
```

Comment:
Such statements are only confusing.

```suggestion
```

Reply:
Why? What confuses you? The function is exactly the same as `ksCut`, except that it doesn't modify the original `KeySet`.

Reply:
Then please write that is the same! Writing that something is similar without describing the differences is confusing.